### DISCUSSION TITLE
Fetch list of authorizations for user(s)

### DISCUSSION THREAD

**Discussion by vkaelin** [zitadel/zitadel#8874] [Q&A] [answered]
Hi 👋 

I was wondering how to get the list of authorizations of an user, to have the list of authorizations by project.

I'm speaking about the authorizations listed here:
<img width="750" alt="image" src="https://github.com/user-attachments/assets/5bf7550f-3a9c-4b6d-b3d8-4b0e14de7283">

I couldn't find an endpoint in the API V2 documentation?

I'm currently using [this users API](https://zitadel.com/docs/apis/resources/user_service_v2/user-service-list-users) to search for users but sadly this information doesn't look to be in the DTO response.
Do I need to do an additional request per user to a specific endpoint to get it? 

Thanks a lot!

**Comment by hifabienne** [comment, marked as answer]
Hi @vkaelin 
You can find this with the following request: https://zitadel.com/docs/apis/resources/mgmt/management-service-list-user-grants

**Reply by vkaelin** [reply]
Thanks a lot! I saw that endpoint but I thought that we had to use V2 instead of V1 because of future deprecation.
